src/main/java/cn/nukkit/Nukkit.java

```diff
@@ -108,6 +109,7 @@ public static void main(String[] args) {
         OptionSpec<String> vSpec = parser.accepts("v", "Set verbosity of logging").withRequiredArg().ofType(String.class);
         OptionSpec<String> verbositySpec = parser.accepts("verbosity", "Set verbosity of logging").withRequiredArg().ofType(String.class);
         OptionSpec<String> languageSpec = parser.accepts("language", "Set a predefined language").withOptionalArg().ofType(String.class);
+        OptionSpec<Void> skipSetupSpec = parser.accepts("skip-setup", "Skip the setup wizard and use default values");
```

Comment:
Maybe we should rename this to skip-setup-agree-license or something like that to address legal concerns